app/Http/Controllers/Administration/UserController.php

```diff
@@ -130,4 +130,48 @@ public function getEmail(): array
 			'email' => AccessControl::user()->email,
 		];
 	}
+
+	/**
+	 * Returns the currently authenticated user.
+	 *
+	 * @return User
+	 */
+	public function getCurrent(): User
+	{
+		return AccessControl::user();
+	}
+
+	/**
+	 * Reset the token of the currently authenticated user.
+	 *
+	 * @return array
+	 *
+	 * @throws \Exception
+	 */
+	public function resetToken(): array
+	{
+		AccessControl::user()->token = strtr(base64_encode(random_bytes(16)), '+/', '-_');
+		AccessControl::user()->save();
+
+		return [
+			'token' => AccessControl::user()->token,
+		];
+	}
```

Comment:
I would prefer to return the entire user object. That is what we do in most cases. Yes I know there are still exceptions to this rule like the unfortunate `getEmail`.

```suggestion
	public function resetToken(): array
	{
		$user = AccessControl::user();
		$user->token = strtr(base64_encode(random_bytes(16)), '+/', '-_');
		$user->save();

		return $user;
	}
```